apps/frontend/src/components/global/checklist/ChecklistRuleInfoBody.vue

```diff
@@ -0,0 +1,61 @@
+<template>
```

Comment:
make severity override text show up in here as well

david says put it into misc data, i personally am not sure if that's the best location so try putting it in different places in the list and see what looks good

Reply:
From our discussion having the content in the rules body info is fine